.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
```

Comment:
`package.json` and `package-lock.json` should not be ignore - they should be pushed to the repository.

Is there a reason we are ignoring it here?